toponetx/datasets/mesh.py

```diff
@@ -32,11 +34,47 @@ def stanford_bunny(cmplex_type="simplicial complex"):
     ValueError
         If cmplex_type is not one of the supported values.
     """
-    if cmplex_type == "simplicial complex":
+    if complex_type == "simplicial complex":
         cpx = SimplicialComplex.load_mesh(DIR + "/bunny.obj")
         return cpx
-    elif cmplex_type == "cell complex":
+    elif complex_type == "cell complex":
```

Comment:
```suggestion
    if complex_type == "cell complex":
```

Reply:
Additionally, let the complex_type be one of:
- "simplicial"
- "cell"
- etc
i.e. avoid to have the " complex" part of the string
--> it's shorter and does not use spaces.